install.sh

```diff
@@ -66,8 +66,12 @@ echo "$(tput bold)$(tput setaf 1)If you need Card Vault, please create master ke
 echo "$(tput bold)$(tput setaf 3)To generate the master key, you can use the utility bundled within \n(https://github.com/juspay/hyperswitch-card-vault)$(tput sgr0)"
 echo "$(tput bold)$(tput setaf 3)If you have cargo installed you can run \n(cargo install --git https://github.com/juspay/hyperswitch-card-vault --bin utils --root . && ./bin/utils master-key && rm ./bin/utils && rmdir ./bin)$(tput sgr0)"
 
+echo "Please input the AES256 compatible master encryption key"
+read -s MASTER_ENC_KEY
+
```

Comment:
Let us update one click deployment files `production.yml` and `single-click.sh` files for the same.